File: Python/48.py
printf = print
tt=1
printf("Esse programa determina o produtório dos números inteiros no intervalo de 1 até n!\n\n")

n = int(input("\n\nDigite ate onde deseja fazer o produtório: "))
i = 1
for i in range(1,n):
    tt = tt * i
    i=i+1
printf(f"\n\nO produtório é de {tt}!")

printf("\n\nObrigado por usar nosso sistema!\n\n")

File: C/48.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(void){
	setlocale(LC_ALL, "Portuguese");
	float i,n,tt;
	printf("Esse programa determina o produtório dos números inteiros no intervalo de 1 até n!\n\n");
	
	printf("\n\nDigite ate onde deseja fazer o produtório: ");
	scanf("%d", &n);
	for(i=1;i<=n;i++){
		tt=tt*i;
	}

	printf ("\n\nO produtório é de %d!", tt);
	
	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}


